butler.py

```diff
@@ -313,9 +311,6 @@ def main():
 
 def _setup():
   """Set up configs and import paths."""
-  appengine_sdk_path = appengine.find_sdk_path()
```

Comment:
Does all other butler commands (other than py_unittest) work without this ?

Reply:
Yes I believe so, tested run_server/run_bot/run/format/lint/bootstrap/integration_tests/package. None of the other ones should be importing from any App Engine SDK paths.